packages/common/src/Enums.js

```diff
@@ -22,11 +22,11 @@ const LiquidationStatesEnum = {
 // Maps the `liquidationStatus` property in the `LiquidationWithdrawn` event to human readable statuses.
 // Note that these are status translations AFTER a withdrawLiquidation method is called
 const PostWithdrawLiquidationRewardsStatusTranslations = {
-  "0": "Liquidation deleted; All rewards have been withdrawn",
```

Comment:
I updated this file to match exactly what is within the smart contracts. @nicholaspai might have an opinion on this but for now I think it's simplest to keep the matching the same.

Reply:
This is better.